pyfstat/mcmc_based_searches.py

```diff
@@ -1233,6 +1233,10 @@ def plot_cumulative_max(self, **kwargs):
 
         if hasattr(self, "search") is False:
             self._initiate_search_object()
+        # Predicted 2F values will be calculated for the parameter space point
+        # in the ".loudest" file generated above,
+        # while the parameters that are explicitly passed here are for the
+        # cumulative 2F calculation on the actual data.
```

Comment:
Problem: the core `plot_twoF_cumulative()` function does not actually currently work as I've documented it here and as this MCMC method seems to assume. Since we're not giving it "signal_parameters", it will just skip the whole PFS block. However, the lowest-level `_calculate_predict_fstat_cumulative()` itself is willing to work with a ".loudest" file instead, so I think just the `if` block in `plot_twoF_cumulative()` needs to be disassembled, and then this method here will get what it wants.